package.json

```diff
@@ -40,9 +41,14 @@
     "mocha-lcov-reporter": "^1.3.0",
```

Comment:
What should really have happened is that line 34 above was replaced with the new version designation.